packages/react-composites/src/composites/MeetingComposite/Strings.tsx

```diff
@@ -29,6 +29,10 @@ export interface MeetingCompositeStrings {
    * Side pane Chat screen title.
    */
   chatPaneTitle: string;
+  /**
+   * New Message label for chat button with notification icon component.
+   */
+  newMessageNotificationLabel: string;
```

Comment:
I think this still needs something that indicates chatButton in the name (https://github.com/Azure/communication-ui-library/pull/1429#discussion_r800926611).
`newMessageNotificationLabel` could be anywhere on the screen -- the doc comment definitely helps but if we were to implement the chat bubbles it's not clear if newMessageNotification is related to the chat button or the chat bubbles (and still conflicts with the message thread "New Messages" button which could be seen as a notification)
![image](https://user-images.githubusercontent.com/2684369/153648373-8f4dd71a-1469-4e30-b434-161f81172c10.png)
![image](https://user-images.githubusercontent.com/2684369/153648477-1a0cc552-edca-4b62-905f-bd5cb965cd89.png)


```suggestion
  chatButtonNewMessageNotificationLabel: string;
```